README.md

```diff
@@ -25,6 +29,9 @@ yarn link
 # Go to the new app created from template.
 cd ../wireline-testing
 yarn link "@wirelineio/serverless-wireline"
+
+# now to install packages
```

Comment:
Nit: please make all comments sentences (i.e., start with capital letter, end with period). Also this doesn't need the "now to" part :).